lib/markdown.js

```diff
@@ -49,9 +49,7 @@ module.exports = function (input, options, cb) {
 
     // inject custom css if exist
     // todo: validate css
-    if (options.customCss) {
-      htmlHeader += '<link rel="stylesheet" href="' + path.resolve(options.customCss) + '">'
-    }
+    options.customCss.forEach(_ => (htmlHeader += '<link rel="stylesheet" href="' + path.resolve(_) + '">'))
```

Comment:
But this isn't the only place lacking line breaks? Plus this is mostly intermediate output for transforming markdown to pdf, I assume?